app/models/storage_profile.rb

```diff
@@ -0,0 +1,5 @@
+class StorageProfile < ApplicationRecord
+  belongs_to :ext_management_system
+  has_many :storage_profile_storages, :dependent  => :destroy
```

Comment:
You mean we keep the relation to the joined table private while keeping the 
``` has_many :storages``` public? 

I tried that and it will fail to do StorageProfile.first.storages as follows,
```
irb(main):008:0> StorageProfile.first.storages
  StorageProfile Load (0.4ms)  SELECT  "storage_profiles".* FROM "storage_profiles" ORDER BY "storage_profiles"."id" ASC LIMIT $1  [["LIMIT", 1]]
  StorageProfile Inst Including Associations (0.1ms - 1rows)
ActiveRecord::StatementInvalid: PG::UndefinedTable: ERROR:  relation "storage_profile_storages" does not exist
LINE 8:                WHERE a.attrelid = '"storage_profile_storages...
                                          ^
```